dotCMS/src/main/java/com/dotmarketing/filters/CookieServletResponse.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * ServletResponseCharacterEncoding.java
+ *
+ * Created on 8/10/2017
+ * @author Will Ezell
+ */
+package com.dotmarketing.filters;
+
+import static com.liferay.util.CookieUtil.COOKIES_HTTP_ONLY;
+
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+
+
+import com.dotmarketing.util.Config;
+
+
+public class CookieServletResponse extends HttpServletResponseWrapper {
+
+
+    public CookieServletResponse(HttpServletRequest request, HttpServletResponse response) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "request". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)